data/json/overmap/specials.json

```diff
@@ -2496,11 +2496,11 @@
       { "point": [ 0, 1, 0 ], "overmap": "dirtroad_north" },
       { "point": [ 0, 2, 0 ], "overmap": "rural_house_north" }
     ],
-    "connections": [ { "point": [ 0, -1, 0 ], "terrain": "road" } ],
+    "connections": [ { "point": [ 0, -1, 0 ], "terrain": "road", "existing": true } ],
     "locations": [ "forest" ],
     "city_distance": [ 5, 25 ],
     "city_sizes": [ 2, 10 ],
-    "occurrences": [ 0, 15 ],
+    "occurrences": [ 1, 5 ],
```

Comment:
This is one where I'd say the original occurences rate is actually probably ok? Some areas might have a lot of rural houses, but there probably isn't a guarantee that every overmap has one?

Agree with the connections change though.